practice-one/src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Todos</title>
+  <link rel="stylesheet" href="./style/main.css">
+  <script type="module" src="./js/app.js"></script>
```

Comment:
script should be placed at the end of body to allo the HTML content to load first before executing scripts.